plugins/techdocs/config.d.ts

```diff
@@ -109,6 +109,45 @@ export interface Config {
             region?: string;
           };
         }
+      | {
+          /**
+           * attr: 'type' - accepts a string value
+           * e.g. type: 'azureStorage'
+           * alternatives: 'azureStorage' etc.
+           * @see http://backstage.io/docs/features/techdocs/configuration
+           */
+          type: 'azureStorage';
```

Comment:
I thought about leaving the name of the service in general, because if at some point in the future the form of implementation changes, it will not break the compatibility of app-config.yml, but if the Backstage team thinks it is better to make this change, I can change no problems

Reply:
Hey! Let's use `azureBlobStorage`. We may have `azureFilesStorage` in future.